formbuilder/components/builder/AddFieldDropdown.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import SchemaField from "react-jsonschema-form/lib/components/fields/SchemaField";
+
+import {Dropdown, MenuItem}  from "react-bootstrap"
+
+import config from "../../config";
+
+export default class AddFieldDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {'fieldList': config.fieldList};
+  }
+
+  handleAddField(field_index, event) {
+    const field = this.state.fieldList[parseInt(field_index)];
+
+    this.props.addField(field);
+  }
+
+  render () {
+    return (
+      <Dropdown title="Add a field" dropup id="split-button-dropup">
```

Comment:
it look's ok on mine.

![galaxy_s4_screenshot](https://cloud.githubusercontent.com/assets/139848/18642195/f7fe8d70-7e9f-11e6-9bd3-678c504e1824.png)